### PR TITLE
Change to deployment apps/v1

### PR DESCRIPTION
**Pull request by gibbleyg**

### CODE REVIEW

File: wordpress-updatable/versions/0.1.3/schema.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
+x-google-marketplace:
+  schemaVersion: v2
+
+  applicationApiVersion: v1beta1
+
+  publishedVersion: '0.1.3'
+  publishedVersionMetadata:
+    releaseNote: >-
+      A new and improved release.
```
Comment (eshiroma) [inline wordpress-updatable/versions/0.1.3/schema.yaml:9]: nit: make this actually descriptive, i.e. "Use supported apiVersions" (which might be more `BugFix` than `Feature`)

Docs: https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/7e45a078237a01ee6af649815ba914646d320aa7/docs/managed-updates.md#top-level-x-google-marketplace-section

File: wordpress-updatable/versions/0.1.2/schema.yaml [hunk 2]
```diff
@@ -11,9 +11,6 @@ x-google-marketplace:
     - Feature
     recommended: true
 
-  managedUpdates:
-    kalmSupported: true
-
```
Comment (eshiroma) [inline wordpress-updatable/versions/0.1.2/schema.yaml:16]: Hm - each release is intended to be immutable, so we should avoid changing 0.1.2

File: wordpress-updatable/versions/0.1.1/schema.yaml [hunk 3]
```diff
@@ -11,9 +11,6 @@ x-google-marketplace:
     - Feature
     recommended: true
 
-  managedUpdates:
-    kalmSupported: true
```
Comment (eshiroma) [inline wordpress-updatable/versions/0.1.1/schema.yaml:15]: Hm - each release is intended to be immutable, so we should avoid changing 0.1.1 and 0.1.2 (they won't be republished)

File: wordpress-updatable/versions/0.1.2/schema.yaml (hunk 2 above)
Comment (gibbleyg) [reply wordpress-updatable/versions/0.1.2/schema.yaml:16]: Ahh I made this change because kalm was causing it to fail for older versions, will add back

**Pull request merged**